pipeline_steps/github.groovy

```diff
@@ -14,15 +14,52 @@ def create_issue(
     sh """#!/bin/bash -xe
       cd ${env.WORKSPACE}
       . .venv/bin/activate
-      python rpc-gating/scripts/ghutils.py create_issue\
+      python rpc-gating/scripts/ghutils.py\
+        --org '$org'\
+        --repo '$repo'\
+        --pat '$pat'\
+        create_issue\
         --tag '$tag'\
         --link '$link'\
+        --label '$label'
+    """
+  }
+}
+
+/**
+ * Add issue link to pull request description
+ *
+ * Pull request commit messages include the issue key for an issue on Jira.
+ * Update the description of the current GitHub pull request with a link to
+ * the Jira issue.
+ */
+void add_issue_url_to_pr(){
+  List org_repo = env.ghprbGhRepository.split("/")
+  String org = org_repo[0]
+  String repo = org_repo[1]
+  Integer pull_request_number = env.ghprbPullId.toInteger()
+  String issue_key = common.get_jira_issue_key(repo_path=repo)
```

Comment:
`repo_path=repo` is not a named parameter its an assignment, which should be ok because assignments return the assigned value in groovy, but not in CPS groovy 😭 😠. The only way of doing named parameters is by accepting a single Map, and then passing a single map.